server/dashboard.go

```diff
@@ -91,10 +91,11 @@ type Summaries struct {
 	PopolazioneAirePreSubentro int `json:"pop_pre_aire"`
 }
 type DashBoardData struct {
-	Geojson   *geojson.FeatureCollection `json:"geojson"`
-	Summaries *Summaries                 `json:"summaries"`
-	Fornitori []FornitoreSums            `json:"fornitori"`
-	Charts    Charts                     `json:"charts"`
+	LastDateTime time.Time                  `json:"lastDateTime"` // "2016-10-21T00:00:00+00:00",
```

Comment:
@LucaPrete are you sure this is properly serialized in json?

Reply:
yes, I've tried it locally.